### DISCUSSION TITLE
Export adif for special durations free text entry for period of date for export

### DISCUSSION THREAD

**Discussion by dl1esz** [N0BOY/FT8CN#85] [Ideas]
Hi,
it would be fine to have a free duration field of data  adif export ( e.g. 05.11.2023 till 07.11.2023 as a export special duration).

By the way fine software for ft8...
B.r. Stefan DL1ESZ

**Comment by bg7yoz** [comment]
hi, @dl1esz 
在0.91版本，是可以自由指定日期导出ADIF文件的。
具体操作如下：
1.通过浏览器访问ft8cn
2.选择“Query the logs”
3.按照您需要的日期查询。
4.选择“Export to ADIF file”
<img width="306" alt="截屏2023-11-15 21 27 25" src="https://github.com/N0BOY/FT8CN/assets/131873746/cd3bd78e-a8ff-4cbf-98a6-6420249de81a">

<img width="553" alt="截屏2023-11-15 21 27 50" src="https://github.com/N0BOY/FT8CN/assets/131873746/880b465f-e311-4c81-935d-114c119e6c88">


BG7YOZ

**Comment by dl1esz** [comment]
Hi,perfect did not recognized this first but no its fine....as i mentioned before very good job !!!grettings from germanyDL1ESZ Stefan 73!Am 15.11.2023 14:38 schrieb bg7yoz ***@***.***>:
hi, @dl1esz
在0.91版本，是可以自由指定日期导出ADIF文件的。
具体操作如下：
1.通过浏览器访问ft8cn
2.选择“Query the logs”
3.按照您需要的日期查询。
4.选择“Export to ADIF file”


BG7YOZ
